bin/milight

```diff
@@ -1,32 +1,199 @@
-#!/usr/bin/env ruby
+#!/usr/lib/env ruby
```

Comment:
Was this intended? /usr/lib/env does not exist on my Mac OS and linux systems.

Reply:
Yeah that should be `bin`. Not sure why it's `lib`. Good spot